### ISSUE TITLE
Video autoplayed?

### ISSUE THREAD

**Issue by marcosybarraa** [sanjayV/ng-image-slider#18]
Hi, 
congrats for your nice and useful work.
If a gallery of videos is created, the user has to press twice to play the video, one that opens the lightbox, and then press again to play. 
Would be great to have "autoplay" as optional input param.

Kind regards

**Comment by sanjayV** [comment]
Yes, their must be option for video autoplay. Thank you for your valuable suggestion :+1: @marcosybarraa 
let me add it..

**Comment by sanjayV** [comment]
Hi @marcosybarraa 
Added **videoAutoPlay** option in new version [@2.0.3](https://www.npmjs.com/package/ng-image-slider)
You just need to use following option 

> [videoAutoPlay]="true"

[Options](https://www.npmjs.com/package/ng-image-slider#api-reference-optional-)
